keyboards/smithrune/iron165r2/f072/rules.mk

```diff
@@ -1,4 +1,24 @@
 # MCU name
 MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # USB Nkey Rollover
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+LTO_ENABLE = no
+ENCODER_ENABLE = no
 BACKLIGHT_ENABLE = yes
 BACKLIGHT_DRIVER = pwm
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

Comment:
Any reason this wasnt added to the f411 rules too?

Reply:
Sorry, didn't think it was necessary for F411. Resolved.